templates/default/start-kibana.sh.erb

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-PID_FILE=<%= node['kibana']['pid_file'] %>
+export PID_FILE=<%= node['kibana']['pid_file'] %>
```

Comment:
why aren't you changing the service name to opensearch_dashboard?

Reply:
I changed it to  opensearch-dashboards - not underscore and plural for dashboards. You don't use underscore in service names.